### DISCUSSION TITLE
Smaller binary (wasm) size?

### DISCUSSION THREAD

**Discussion by stereobooster** [duckdb/duckdb-wasm#1469] [Ideas]
I assume, that by default DuckDB-Wasm includes all features and that is why binary is big (~17mb). Is there a way to make builds containing only subset of features? For example, if I don't need S3 support or I don't need parquet support.

For comparison,  sql-wasm.wasm is 599K (from [sql.js](https://github.com/sql-js/sql.js))

**Comment by choucavalier** [comment]
any answer to this? i currently need a huge number of megabytes while i only need parquet support

**Reply by domoritz** [reply]
If you just need Parquet, check out https://www.npmjs.com/package/parquet-wasm.

**Comment by MengLinMaker** [comment]
This would be appreciated as some infrastructure providers like CloudFlare limit asset sizes to 25mb

**Reply by tarangrathod** [reply]
i tried brotli compression using cloudflare i can see the current binary is 35mb which got  squezed to 4.8 mb ! i tried with nginx too used brotli compression its working best on wasm binary as compaired to zstandard and gzip
